common/src/Model/GameFactory.cpp

```diff
@@ -47,7 +47,7 @@
 
 namespace TrenchBroom {
     namespace Model {
-        GameFactory::~GameFactory() {
+        void GameFactory::writeConfigs() {
```

Comment:
I think it should be called `saveAllConfigs` and it should be moved to just above `saveConfig(const std::string&)`.